README.md

```diff
@@ -1,33 +1,82 @@
 
 
-We believe that all of us want to - and can - make a difference to the Guardian.
+**Working as a team**
+
+We believe that the team is everything and our lowest common unit, set up with the necessary skills to succeed.
 
->We choose to work here because we believe in what the Guardian stands for. We're excited at contributing to the Guardian's future. 
+_Behaviours_:
+ - We fail and succeed together as a team.
+ - We are independent, accountable and share responsibility. We conduct ourselves with humility, kindness and a calm manner. We debate ideas, not people and we share constructive feedback openly. 
+ - We take the time needed to listen to and help our colleagues with any problem they might have from anywhere within the business.
+ - We value the experience of people in the team and the different constraints within their lives. 
+ - We celebrate success and positive behaviour, recognizing everyone’s contribution while treating every moment including tasks and failures as a learning opportunity.
 
-We believe in treating everyone we encounter with respect.
+**We are data informed** 
 
->Whether meeting physically or electronically, whether friend or stranger, whether employee, customer or commenter, we respect the opinion of others and are gracious in our interactions.
+We believe that to be responsible in our impact we must be informed by data.
 
-We are respectful of different skill sets and levels of ability.
->We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
+_Behaviours_: 
+ - We base our decisions on analytics and experiments to reach consensus having heard everyone’s voice. 
+ - We measure our success through data analysis and not through the highest paid person's opinion or the loudest voice.
 
-We believe we are empowered to meet the needs of people we encounter across the Guardian.
+**Empowerment**
 
->This behaviour is consistently encouraged across all levels of engineering.
+We believe that all of us want to - and can - make a difference to the Guardian.
 
-We believe that embracing technical innovation and excellence can help us make a difference
+_Behaviours_: 
+  - Meeting the needs of people we encounter across the Guardian is consistently encouraged across all levels of engineering.
+  - We trust engineers to build the right things, collaborating closely and transparently with other teams.
+  - We own the problem, not the spec. We don’t wait for decisions to be made for us, we are an empowered engineering function and you are an empowered engineer. We use that power responsibly as we are conscious some decisions may need to be signed off and work collaboratively with our stakeholders.
```

Comment:
This should probably be broken down into smaller points. Some of them are also beliefs rather than behaviours

Reply:
Any suggestion on how you would rewrite it?